packages/office-ui-fabric-react/src/utilities/positioning/positioning.ts

```diff
@@ -263,6 +263,32 @@ function _flipToFit(
   };
 }
 
+/**
+ * Flips only the alignment edge of an element rectangle. This is used instead of nudging the alignment edges into position,
+ * when alignPerfectlyWithTarget is specified.
+ * Returns the initial elementEstimate if the flipping does not work.
+ * @param elementEstimate
+ * @param target
+ * @param bounding
+ * @param gap
+ */
+function _flipAlignmentEdgeOnly(elementEstimate: IElementPosition, target: Rectangle, bounding: Rectangle, gap: number): IElementPosition {
+  const { alignmentEdge, targetEdge, elementRectangle } = elementEstimate;
+  if (alignmentEdge !== undefined) {
```

Comment:
alignmentEdge won't ever actually be null or undefined at this point. I'll need to make that more clear in comments.